test/alternative_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "../src/lib/update-alternatives/model/alternative"
+require_relative "../src/lib/update-alternatives/model/choice"
```

Comment:
these two not needed, simple use what other modules uses...so set properly Y2DIT in helper and require yast and then you can use common 
```
require "update-alternatives/model/alternative"
require "update-alternatives/model/choice"
```